docs/tutorial.md

```diff
@@ -82,13 +82,22 @@ If rescheduling is enabled, the due dates of cards will be immediately changed.
 
 ![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/fe61aaa6-cecb-4476-9ed2-9db05b63c7de)
 
+### Ignore reviews before
+
+In Anki 24.04, a feature to exclude review history until a specific date from optimization has been added. Only reviews after the selected date will be used by the optimizer, as well as during evaluation.
+
+![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/83031600/1d53d964-2136-43b2-977f-5116141c38f9)
+
+It is useful if you have been misusing Hard. Hard should be used as a passing grade, not a failing grade. You can also use this feature if your learning habits have changed very significantly over time, however, FSRS can likely adapt to those changes.
```

Comment:
```suggestion

This can be useful if you imported someone else's scheduling data, or if you have been misusing Hard. Hard should be used as a passing grade, not a failing grade. This feature can also be helpful if your learning or rating habits have changed significantly over time.
```